tests/integration/ha/continuous_writes.py

```diff
@@ -269,7 +276,7 @@ def _bulk(client: OpenSearch, write_value: int) -> None:
         """Bulk Index group of docs."""
         data = []
         for i in range(100):
-            val = (100 * write_value) + i
+            val = write_value + i
```

Comment:
What we had was correct, we're ensuring that this **only** gets incremented if `write_value` has been successfully incremented.